app/views/locationPickerMap.html

```diff
@@ -9,13 +10,19 @@ <h3 class="modal-title">Select Ticketed Location</h3>
     <div ng-if="ctrl.mousedOverMunicipality != null">Municipality: {{ctrl.mousedOverMunicipality.name}}</div>
     <div style="margin-top: 20px;" ng-if="ctrl.selectedMunicipalities.length > 0">
       <label>Select Cities</label><br/>
-      <span ng-repeat="municipality in ctrl.selectedMunicipalities">
-        {{municipality.name}}<span ng-if="$index < ctrl.selectedMunicipalities.length - 1">, </span>
-      </span>
+      <form-field>
+        <ui-select id="selectedMunicipalityFromMap" ng-disabled="true" multiple name="selectedMunicipality" register-custom-form-control required restrict-number-of-municipalities="{{ctrl.maxMapMunicipalities}}" displayName="Municipality"  ng-model="ctrl.selectedMunicipalities">
```

Comment:
I still think you should be able to use ng-max on here vs writing a whole new validator directive. But it's fine as-is and good for you to learn it.